src/Leaderboard.ts

```diff
@@ -164,7 +164,21 @@ export default class Leaderboard {
 
       for (let i = 1; i < 26; i++) {
         if (i in member.completion_day_level) {
-          if (member.completion_day_level[i][2]) stars += ':star:'; // part 1 and 2 are complete
+          if (member.completion_day_level[i][2]){
+            // part 1 and 2 are complete
+            const completed_ts: number = Number(member.completion_day_level[i][2].get_star_ts)*1000;
+
+            // check if star is younger than an hour
+            if (completed_ts - (now.getTime() - 3600000/*one hour*/) > 0){
+              // star is younger than one hour
+              stars += ':star2:';
+              
+            } else if(completed_ts-10800000 < this.getDay(now, i).getTime()) { //TODO: this doesn't work
+              // member completed day in under 3 hours
```

Comment:
Isn't this TODO outdated? This whole request is about that feature and here it says it doesn't work

Reply:
yeah, that TODO is outdated.... forgot to remove it